sdk/eventhubs/azure-messaging-eventhubs/README.md

```diff
@@ -429,6 +433,7 @@ Guidelines](./CONTRIBUTING.md) for more information.
 
 <!-- Links -->
 [amqp_transport_error]: https://docs.oasis-open.org/amqp/core/v1.0/os/amqp-core-transport-v1.0-os.html#type-amqp-error
+[aad-authorization]: https://docs.microsoft.com/azure/event-hubs/authorize-access-azure-active-directory
```

Comment:
Just for consistency, use `underscore` instead of `hyphen` for links. 
```suggestion
[aad_authorization]: https://docs.microsoft.com/azure/event-hubs/authorize-access-azure-active-directory
```